Automattic/jetpack
# Add tracks events for viewing upgrade opportunities

Add tracks events for viewing the Getting Started page, and for seeing the upgrade CTA on the dashboard.

#### Changes proposed in this Pull Request:
* Add tracks events for viewing the Getting Started page, and for seeing the upgrade CTA on the dashboard.

#### Other information:

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
Yes, it adds two tracks events.

#### Testing instructions:
* View the Getting Started page by installing the plugin on a new site, ensure the tracks event fires correctly.
* View the Boost dashboard w/o any upgrades to ensure the tracks event for the CTA impressions fires correctly.
